docs/source/en/using-diffusers/inference_with_tcd_lora.md

```diff
@@ -0,0 +1,433 @@
+<!--Copyright 2024 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+-->
+
+[[open-in-colab]]
+
+# Performing inference with TCD-LoRA
+
+Trajecotroy Consistency Distillation (TCD) enables the model to generate higher quality, more detailed images with fewer steps. Additionally, TCD demonstrates superior performance even under conditions of high NFEs.
+
+From the [Official Project Page](https://mhh0318.github.io/tcd/), the major merit of TCD can be outlined as follows:
+
+> ***Better than Teacher:*** TCD maintains superior generative quality at both low NFEs and high NFEs, even exceeding the performance of DPM-Solver++(2S) with origin SDXL. It is worth noting that there is no additional discriminator or LPIPS supervision included during training.
```

Comment:
The reader doesn't yet know what NFE is. Also, it's worth hyperlinking DPM-Solver++(2S).